src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -347,6 +351,12 @@ def load_arguments(self, _):
         c.argument('enable_hotpatching', arg_type=get_three_state_flag(), help='Patch VMs without requiring a reboot. --enable-agent must be set and --patch-mode must be set to AutomaticByPlatform', min_api='2020-12-01')
         c.argument('platform_fault_domain', min_api='2020-06-01',
                    help='Specify the scale set logical fault domain into which the virtual machine will be created. By default, the virtual machine will be automatically assigned to a fault domain that best maintains balance across available fault domains. This is applicable only if the virtualMachineScaleSet property of this virtual machine is set. The virtual machine scale set that is referenced, must have platform fault domain count. This property cannot be updated once the virtual machine is created. Fault domain assignment can be viewed in the virtual machine instance view')
+        c.argument('security_type', arg_type=get_enum_type(['TrustedLaunch']), min_api='2020-12-01',
```

Comment:
just curious, is there any relationship among the 3 params?

Reply:
https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch
Secure boot and vTPM are features of trusted launch.